### PR TITLE
use docker anchors

### PR DESCRIPTION
**Pull request by conorbros**
large nit fix

### CODE REVIEW

File: shotover-proxy/example-configs/cassandra-cluster-multi-rack/docker-compose.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ networks:
 
 services:
   cassandra-one:
-    image: bitnami/cassandra:4.0.6
+    image: bitnami/cassandra:4.0
```
Comment (rukai) [inline shotover-proxy/example-configs/cassandra-cluster-multi-rack/docker-compose.yml:14]: is this change intentional?

Comment (conorbros) [reply shotover-proxy/example-configs/cassandra-cluster-multi-rack/docker-compose.yml:14]: This set off a rampage of adding anchors wherever I could 😂

File: shotover-proxy/example-configs/redis-cluster-dr/docker-compose.yml [hunk 2]
```diff
@@ -77,67 +67,55 @@ services:
...
-    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    image: *image
     ports:
       - "2120:6379"
     volumes:
       - redis-cluster_data-0-dr:/bitnami/redis/data
-    environment:
-      - 'REDIS_PASSWORD=shotover'
-      - 'REDIS_NODES=redis-node-0-dr redis-node-1-dr redis-node-2-dr redis-node-3-dr redis-node-4-dr redis-node-5-dr'
+    environment: *environment
```
Comment (rukai) [inline shotover-proxy/example-configs/redis-cluster-dr/docker-compose.yml:75]: this should be redis-node-x-dr instead of redis-node-x

File: shotover-proxy/example-configs/cassandra-cluster-multi-rack/docker-compose.yml (hunk 1 above)
Comment (rukai) [reply shotover-proxy/example-configs/cassandra-cluster-multi-rack/docker-compose.yml]: :tada:

**Review by rukai — APPROVED** [review]
Very nice!

I often make mistakes when im quickly testing something where I only update one of the containers, so this is much appreciated.

**Review by benbromhead — APPROVED** [review]
nice!
